src/agent.c

```diff
@@ -717,6 +717,12 @@ agent_transact_pageant(LIBSSH2_AGENT *agent, agent_transaction_ctx_t transctx)
         }
         memcpy(transctx->response, p + 4, transctx->response_len);
     }
+    else {
+        UnmapViewOfFile(p);
+        CloseHandle(filemap);
+        return _libssh2_error(agent->session, LIBSSH2_ERROR_AGENT_PROTOCOL,
+                              "no response from pageant");
```

Comment:
```suggestion
                              "pagent did not handle message");
```
We can go with that, also to have another go with the CI.
For pedandic++, I'd suggest dropping the contraction:
"pagent did not handle message"